configuration/configuration.go

```diff
@@ -37,12 +38,17 @@ const (
 
 	// DefaultIdleTimeout before closing an idle connection.
 	DefaultIdleTimeout = 180 * time.Second
+
+	// DefaultGraceTimeout controls how long Traefik serves pending requests
+	// prior to shutting down.
+	DefaultGraceTimeout = 10 * time.Second
 )
 
 // GlobalConfiguration holds global configuration (with providers, etc.).
 // It's populated from the traefik configuration file passed as an argument to the binary.
 type GlobalConfiguration struct {
-	GraceTimeOut              flaeg.Duration          `short:"g" description:"Duration to give active requests a chance to finish before Traefik stops"`
+	LifeCycle                 *LifeCycle              `description:"Timeouts influencing the server life cycle"`
```

Comment:
Do you expect other settings than timeouts inside of the `LifeCycle` struct? If not do we want to rename it to `LifeCycleTimeouts` for consistency reasons?

Reply:
We might, I don't know.

The likely more important point for me is that the other approach stutters: `LifeCycleTimeouts.GraceTimeOut` has one timeout too many, so it should either be removed from the struct name or the parameters IMHO. I decided for the former as that allows us to extend the struct in the future without the need for another name update.

Reply:
I see your point. SGTM.